mvds.md

```diff
@@ -139,6 +140,10 @@ The record of the type `Type` SHOULD be retransmitted every time `Send Epoch` is
 
 > ***NOTE:** We do not retransmission `ACK`s as we do not know when they have arrived, therefore we simply resend them every time we receive a `MESSAGE`.*
 
+## Formal Specification
+
+MVDS has been formally specified using TLA+: <https://github.com/vacp2p/formalities/tree/master/MVDS>.
```

Comment:
do we want to version this as well?

and what properties it verifies (or link to it, e.g. in that README)

Reply:
@oskarth I think we should just state in the other README which version it implements. Yeah I will state which properties it verifies in the other repository.